torch/testing/_internal/common_fsdp.py

```diff
@@ -1118,7 +1150,7 @@ def setUp(self):
 
     @property
     def world_size(self):
-        return min(torch.cuda.device_count(), 8) if torch.cuda.is_available() else 4
```

Comment:
~todo- is this actually unchanged in the new formula?~

as long as `TEST_CUDA := torch.cuda.is_available()`, then `DEVICE_COUNT := torch.cuda.device_count()` so this appears equivalent.